### PR TITLE
Rubocop

### PR DESCRIPTION
**Pull request by packrat386**
💚 

There's a `.rubocop.yml` in the repo, but it's not a test dependency and the current code is violating it. This fixes the current violations (most of which were auto-correctable) and adds rubocop as a required test on travis. Also adds tests for Rails 5.

Thanks to @fwininger for the help.

### CODE REVIEW

**Comment by kjg** [comment]
Seems like there is some sort of parse error with travis, the last test's gemfile is all of them somehow

**Comment by fwininger** [comment]
Thank you @packrat386 ! :+1: for the pull request

**Comment by fwininger** [comment]
:+1: for the last commit @packrat386

File: .travis.yml [hunk 1]
```diff
@@ -22,20 +30,38 @@ matrix:
...
-    - rvm: 2.2.1
+    - rvm: 2.3.1
       gemfile: gemfiles/rails_31.gemfile
-    - rvm: 2.2.1
+    - rvm: 2.3.1
       gemfile: gemfiles/rails_32.gemfile
+  include:
+    - rvm: 2.3.1
+      gemfile: gemfiles/rails_5.gemfile
+      env: TEST_SUTE=rubocop
```
Comment (kjg) [inline .travis.yml]: looks like a typo here. TEST_SUITE

**Pull request merged**